src/main/java/dev/jbang/cli/Info.java

```diff
@@ -239,11 +235,38 @@ ProjectBuilder createProjectBuilder() {
 @CommandLine.Command(name = "tools", description = "Prints a json description usable for tools/IDE's to get classpath and more info for a jbang script/application. Exact format is still quite experimental.")
 class Tools extends BaseInfoCommand {
 
+	@CommandLine.Option(names = {
```

Comment:
not a fan of --select the name; but i don't have a better one - lets at least get docs for these changes added.

Reply:
It's a _very_ minor and technical feature and we don't even mention `jbang info` outside the CLI docs. So I'm not sure what kind of docs do you want for this that won't already be in the generated CLI docs?

Reply:
Well, like I said "I'm not sure what kind of docs do you want for this that won't already be in the generated CLI docs"?

Reply:
well to actually list it in the main docs that you can do `jbang info jarpath GAV`

Reply:
Aah, ok, I thought you meant I had to mention that `--select` option somehow. But you're talking about `info` in general.So sure, I agree, but that would be unrelated to this PR :-)